apps/odk_publish/views.py

```diff
@@ -144,3 +146,75 @@ def form_template_publish_next_version(request: HttpRequest, odk_project_pk, for
     version = form_template.create_next_version(user=request.user)
     messages.add_message(request, messages.SUCCESS, f"{version} published.")
     return HttpResponse(status=204)
+
+
+@login_required
+def app_user_export(request, odk_project_pk):
+    """Exports AppUsers to a CSV or Excel file.
+
+    For each user in the current project, there will be "id", "name", and "central_id"
+    columns. Additionally there will be a column for each TemplateVariable related to
+    the current project.
+    """
+    resource = AppUserResource(request.odk_project)
+    form = AppUserExportForm([resource], data=request.POST or None)
+    if request.method == "POST" and form.is_valid():
+        export_format = form.cleaned_data["format"]
+        dataset = resource.export(
+            request.odk_project.app_users.prefetch_related("app_user_template_variables")
+        )
+        data = export_format.export_data(dataset)
+        filename = f"app_users_{odk_project_pk}_{localdate()}.{export_format.get_extension()}"
+        return HttpResponse(
+            data,
+            content_type=export_format.get_content_type(),
+            headers={"Content-Disposition": f'attachment; filename="{filename}"'},
+        )
+
+    context = {
+        "breadcrumbs": Breadcrumbs.from_items(
+            request=request,
+            items=[("App Users", "app-user-list"), ("Export", "app-users-export")],
+        ),
+        "form": form,
+    }
+    return render(request, "odk_publish/app_user_export.html", context)
+
+
+@login_required
+def app_user_import(request, odk_project_pk):
+    """Imports AppUsers from a CSV or Excel file.
+
+    The file is expected to have the same columns as a file exported using the
+    app_user_export view above. New AppUsers will be added if the "id" column is blank.
+    If a AppUserTemplateVariable column is blank and it exists in the database,
+    it will be deleted.
+    """
+    # TODO: Add a preview page after a file is uploaded, similar to imports in Django Admin
+    resource = AppUserResource(request.odk_project)
+    form = AppUserImportForm([resource], data=request.POST or None, files=request.FILES or None)
+    if request.POST and form.is_valid():
+        result = resource.import_data(
+            form.dataset, use_transactions=True, rollback_on_validation_errors=True
+        )
+        if not (result.has_validation_errors() or result.has_errors()):
+            return redirect("odk_publish:app-user-list", odk_project_pk=odk_project_pk)
```

Comment:
Adding a success message might be nice for the user to know the import was successful